atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsDeleteBenchmarks.java

```diff
@@ -36,6 +39,18 @@ private Object doDelete(RegeneratingTable<Multimap<Cell, Long>> table) {
         return table.getTableCells();
     }
 
+    private Object doDeleteRange(ConsecutiveNarrowTable table, int numBatches) {
+        Iterable<RangeRequest> rangeRequests;
```

Comment:
You can use `? :` syntax to join the declaration and assignment. I prefer this way, but wouldn't block on it.